test/Marketplace/initialize.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect } from "chai";
+import { ethers, upgrades } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { Marketplace, Marketplace__factory } from "../../types/typechain-types";
+import { Contract } from "ethers";
+
+describe("Method: initialize", () => {
+  let deployer: SignerWithAddress;
+  let MarketplaceInstance: Marketplace__factory;
+  let MarketplaceBeacon: Contract;
+  let marketplace: Marketplace;
+
+  before(async () => {
+    [deployer] = await ethers.getSigners();
+
+    MarketplaceInstance = await ethers.getContractFactory("Marketplace");
+    MarketplaceBeacon = await upgrades.deployBeacon(MarketplaceInstance);
+    marketplace = (await upgrades.deployBeaconProxy(
+      MarketplaceBeacon,
+      MarketplaceInstance,
+      []
+    )) as Marketplace;
+  });
+
+  describe("When all parameters correct", () => {
+    it("should be deployed", async () => {
+      expect(marketplace.address).to.be.properAddress;
+    });
+
+    it("should admin get DEFAULT_ADMIN_ROLE", async () => {
+      const defaultAdminRole = await marketplace.DEFAULT_ADMIN_ROLE();
+      const roleStatus = await marketplace.hasRole(defaultAdminRole, deployer.address);
+      expect(roleStatus).to.true;
+    });
```

Comment:
Нужно придерживаться одного стиля кода.
Например как тут отделять проверку ``expect(roleStatus).to.true;`` от получения данных.


```suggestion
    it("should admin get DEFAULT_ADMIN_ROLE", async () => {
      const defaultAdminRole = await marketplace.DEFAULT_ADMIN_ROLE();
      const roleStatus = await marketplace.hasRole(defaultAdminRole, deployer.address);

      expect(roleStatus).to.true;
    });
```